rst2pdf/styles.py

```diff
@@ -610,7 +611,16 @@ def readStyle(self, ssname):
         fname = self.findStyle(ssname)
         if fname:
             try:
-                return rson_loads(open(fname).read())
+                # Is it an older rson/json stylesheet with .style extension?
+                root_ext = os.path.splitext(fname)
+                if root_ext[1] == ".style":
+                    log.warn(
```

Comment:
`log.warn` is deprecated. Use `log.warning`. https://stackoverflow.com/a/15655674

Reply:
Oops! Good catch.